packages/waci-pex/__tests__/local-agent.test.tsx

```diff
@@ -0,0 +1,27 @@
+import {getConfig} from '@veramo/cli/build/setup'
+import {createObjects} from '@veramo/cli/build/lib/objectCreator'
+
+jest.setTimeout(30000)
+
+import SsiQrCodeProviderLogic from './shared/ssi-qr-code-provider-logic'
```

Comment:
import does not seem correct, check with the other plugins

Reply:
must be a bad merge issue